docs/pages/server-access/rbac.mdx

```diff
@@ -178,6 +178,8 @@ spec:
       # Optional: Session recording mode for SSH sessions.
       # If not set, the value set on default will be used.
       ssh: best_effort|strict
+    # require_session_mfa require per-session MFA for any owner of this role
```

Comment:
Is it optional? Is there a default value? Also:

```suggestion
    # require_session_mfa requires per-session MFA for any owner of this role
```

Reply:
Yes, it's optional, please see update